src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT = 1 << 4;
+    private static final float LOAD_FACTOR = 0.75f;
+    private LinkedList<Entry<K,V>>[] buckets;
+    private V null1;
+    private int size;
+
+    public MyHashMap() {
+        buckets = new LinkedList[DEFAULT];
+        for (int i = 0; i < DEFAULT; i++) {
+            buckets[i] = new LinkedList<>();
+        }
+    }
+
+    private static class Entry<K, V> {
+        private K key;
+        private V value;
+
```

Comment:
Remove the getters and setters for the inner class 'Entry' as they are redundant. You have direct access to the private fields of the inner class within the outer class.